### PR TITLE
Add libvips to Development Dependency guide [ci skip]

### PR DESCRIPTION
**Pull request by skipkayhil**
### Summary

Package listings:

macos/brew - https://formulae.brew.sh/formula/vips
debian/apt - https://packages.ubuntu.com/jammy/libvips42
fedora     - https://packages.fedoraproject.org/pkgs/vips/vips/
arch linux - https://archlinux.org/packages/community/x86_64/libvips/
freebsd    - https://cgit.freebsd.org/ports/tree/graphics/vips

### Other Information

I was able to test installing all of these except FreeBSD, would definitely appreciate someone correcting me if I got that one wrong.

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -14,3 +14,4 @@ cask "xquartz"
 brew "mupdf"
 brew "poppler"
 brew "imagemagick"
```
Comment (p8) [inline Brewfile:16]: Should "imagemagick" be removed if vips is the default?

Comment (SkipKayhil) [reply Brewfile:16]: ImageMagick functionality is still being tested here: https://github.com/rails/rails/blob/35d8890c92b931cecf323d4bdd7e318f20326926/activestorage/test/analyzer/image_analyzer/image_magick_test.rb, so I don't think we should remove it. (I also tried running this file without ImageMagick binaries `convert` and `identify` and 3 of the tests fail)

Comment (p8) [reply Brewfile:16]: Ah these changes are for working on Rails itself 👍

**Comment by p8** [comment]
Thanks @SkipKayhil !

**Pull request merged**